tests/test_qmakebuild/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+#set(TARGET wellknowntypes_test)
+project(qmakebuild_test)
+
+get_target_property (QT_QMAKE_EXECUTABLE Qt5::qmake IMPORTED_LOCATION)
```

Comment:
CMake accepts using the target as the first argument of "COMMAND" without expanding its path. If it's the second or subsequent argument you may use $<TARGET_FILE:Qt5::qmake> genex directly without getting the imported location.

Reply:
I haven't been able to make this work yet.  Specifically, I tried
```
COMMAND ${TARGET_FILE:Qt5::qmake} -o QMakefile
```
and got 
```
Invalid character (':') in a variable name: 'TARGET_FILE'
```

Reply:
genexes use `<` and `>` instead of `{` and `}`